components/create_post/create_post.tsx

```diff
@@ -802,6 +804,14 @@ class CreatePost extends React.PureComponent<Props, State> {
         }
     }
 
+    scrollDownTexbox = () => {
+        setTimeout(() => {
```

Comment:
It doesn't work without the setTimeout(). I didn't find why but it seem to related to react lifecycle with child ref. I think maybe the draft text is loaded after we update scrollTop. So is value is still 0.

Reply:
let me take a look at this shortly

Reply:
@M-ZubairAhmed Have you had a chance to look at why this might be needed? I wouldn't be surprised if it's something like the resizing logic for the post textbox doesn't happen until after its own `setTimeout` so this also needs to be delayed.